scripts/build-controller.sh

```diff
@@ -85,6 +85,14 @@ pushd services/$SERVICE/apis/$ACK_GENERATE_API_VERSION 1>/dev/null
 controller-gen object:headerFile=$TEMPLATES_DIR/boilerplate.txt paths=./...
 popd 1>/dev/null
 
+echo "Generating custom resource definitions for $SERVICE"
+pushd services/$SERVICE/apis/$ACK_GENERATE_API_VERSION 1>/dev/null
```

Comment:
I suppose you could have avoided push/popping directories again and just placed this in the block above, but this is all good :)